src/test/java/api/loans/DeclareLostAPITests.java

```diff
@@ -453,6 +458,38 @@ public void cannotDeclareItemLostTwice() {
       hasParameter("itemId", itemId.toString()))));
   }
 
+  @Test
+  public void shouldNotChargeOverdueOnCheckinWhenItemLostAndRefundFeePeriodPassed() {
+    ItemResource item = itemsFixture.basedUponSmallAngryPlanet();
+
+    IndividualResource overduePolicy = overdueFinePoliciesFixture.facultyStandard();
+    IndividualResource lostItemPolicy = lostItemFeePoliciesFixture.ageToLostAfterOneWeek();
+
+    policiesActivation.use(PoliciesToActivate.builder()
+      .lostItemPolicy(lostItemPolicy)
+      .overduePolicy(overduePolicy)
+    );
```

Comment:
What is the intent of putting the final bracket on a separate line? Is it to give a similar shape to a code block?

```suggestion
    policiesActivation.use(PoliciesToActivate.builder()
      .lostItemPolicy(lostItemPolicy)
      .overduePolicy(overduePolicy));
```

Reply:
Likely just a formatting bug-I write and rewrote many of these tests several times.